com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineMaterialUpgrader.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
Is this needed?

Reply:
Removed in 25554645a928746093a54734dbfc3f3143cd0208.